source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,218 @@
+---
+title: Direct Debit
+weight: 75
+---
+
+# GOV.UK Pay Variable Direct Debit
+
+Before reading this section, you should be familiar with the
+[__Making payments__ section](/payment_flow_overview/#payment-flow).
+
+If you have already [integrated with the GOV.UK Pay
+API](/integrate_with_govuk_pay/#how-to-integrate-with-the-gov-uk-pay-api), you
+can start using Direct Debit quickly. You should:
+
+* request a GOV.UK Pay Direct Debit test account [from our
+support team](/support_contact_and_more_information/#contact-us).
+* add support for the Direct Debit endpoints to your existing GOV.UK Pay
+integration
+* sign in to your GOV.UK Pay Direct Debit account and follow the instructions
```

Comment:
Will this be obvious to users and will the account be fully available at the point of shipping this documentation?

Reply:
they would need to find it via the 'My services' page. we know that sometimes users struggle to find this page, but we could deal with this through sending a (templated) email with steps to 1. find Pay DD account and 2. setup GC merchant account once we've set up a DD gateway account - as we'd need to notify them that it's ready.